test.js

```diff
@@ -136,5 +136,25 @@ assert.deepStrictEqual(
   {}
 );
 
+assert.deepStrictEqual(
+  editorconfigToPrettier({
+    indent_style: "space",
+    indent_size: 2,
+    max_line_length: "unset"
+  }),
+  {
+    useTabs: false,
+    tabWidth: 2
+  }
+);
+
+assert.deepStrictEqual(
+  editorconfigToPrettier({
+    useTabs: false,
+    tabWidth: "unset"
```

Comment:
What's the intent behind this test? It looks like the keys here are already Prettier-style (camelCase) rather than EditorConfig-style (snake_case)...

Reply:
Obviously that's a mistake 🤦‍♂️ 

Can't remember now, I'll just remove